.github/workflows/daily-nightly-jobs.yml

```diff
@@ -76,14 +85,6 @@ jobs:
       - name: wait for ceph to be ready
         run: tests/scripts/github-action-helper.sh wait_for_ceph_to_be_ready "mon,mgr,osd,mds,rgw,rbd_mirror,fs_mirror" 1
 
-      - name: teardown minikube and docker
```

Comment:
I guess we can also remove it at the end of the test would not have any problem.
Or instead of stating at the test it would be good if we move only this to the end

Reply:
IMO, it makes more sense to first clean and install